gradoop-flink/src/test/java/org/gradoop/flink/io/impl/csv/CSVDataSinkTest.java

```diff
@@ -39,15 +40,27 @@ public void testWrite() throws Exception {
       .getDatabase()
       .getDatabaseGraph(true);
 
-    DataSink csvDataSink = new CSVDataSink(tmpPath, getConfig());
-    csvDataSink.write(input, true);
-
-    getExecutionEnvironment().execute();
+    checkCSVWrite(tmpPath, input);
+  }
 
-    DataSource csvDataSource = new CSVDataSource(tmpPath, getConfig());
-    LogicalGraph output = csvDataSource.getLogicalGraph();
+  @Test
+  public void testWriteWithDifferentPropertyTypes() throws Exception {
```

Comment:
Clone this test for `IndexedCSVDataSinkTest` to make sure this functionality works with this sink too.